openff/bespokefit/cli/executor/submit.py

```diff
@@ -225,8 +297,8 @@ def _submit_cli(
     console = rich.get_console()
     print_header(console)
 
-    if (input_file_path is not None and molecule_smiles is not None) or (
-        input_file_path is None and molecule_smiles is None
+    if (input_file_path and molecule_smiles) or (
+        not input_file_path and not molecule_smiles
     ):
```

Comment:
Hmm I wonder if we can just drop this check + error now?

Reply:
Yeah, I think it would be safe to allow both input types.